README.md

```diff
@@ -324,8 +324,13 @@ Additional options can be prepended/appended to an `Options`:
 use Portavice\Bladestrap\Support\Options;
 
 $options = Options::fromModels(User::query()->get(), 'name')
-    ->prepend('all', '') // adds an option with empty value at the start
-    ->append('last', 'last'); // adds at the end
+    ->sortAlphabetically() // call sort for current options
+    ->prepend('all', '') // adds an option with empty value before first option
+    ->append('last', 'last') // adds an option after the last option
```

Comment:
I think the `append('last', 'last')` example is a bit iffy.

Is the first or the second argument the new value? Which one is the position?